manifests/weave/boarnet/bns/genesis.json

```diff
@@ -0,0 +1,458 @@
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "AD102CF85A366D726BBBCA54DBD3818B6278A66A",
+        "//name": "Ingrid",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "0EDAA1225975A3A64BC7216F9429A3EA34A757D9",
+        "//name": "Jonny",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+        "//name": "Cloe",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "cond:gov/rule/0000000000000002",
+        "//name": "reward fund",
+        "coins": [
+          {
+            "whole": 1,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+        "//name": "faucet",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "B1DE2F431945B8D7CE20DDD5F43FDA9609E9D7B8",
+        "//name": "loadtester",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      }
+    ],
+    "currencies": [
+      {
+        "ticker": "IOV",
+        "name": "Main token of this chain"
+      }
+    ],
+    "multisig": [
+      {
+        "activation_threshold": 1,
+        "admin_threshold": 1,
+        "participants": [
+          {
+            "signature": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+            "//name": "Cloe",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+    "update_validators": {
+      "addresses": [
+        "cond:gov/rule/0000000000000004"
+      ]
+    },
+    "distribution": [
+      {
+        "//name": "unused - owned by economic committee",
+        "admin": "cond:gov/rule/0000000000000002",
+        "destinations": [
+          {
+            "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+
+    "escrow": [
+      {
+        "//name": "guarantee fund",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1000000
+          }
+        ],
+        "arbiter": "cond:gov/rule/0000000000000002",
+        "destination": "cond:gov/rule/0000000000000002",
+        "//destination": "reward fund - cash account",
+        "source": "0000000000000000000000000000000000000000",
+        "timeout": "2999-12-31T00:00:00Z"
+      },
+      {
+        "//name": "vesting first tranche",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1234567
+          }
+        ],
+        "arbiter": "0000000000000000000000000000000000000000",
+        "destination": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17",
+        "source": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17",
+        "timeout": "2019-08-28T00:00:00Z"
```

Comment:
This is expired already. If we want to use this as a template for the next network let's add some years

Reply:
The idea behind this is to enforce the vesting periods on founders' funds by using an escrow.  `boarnet` was launched on the 26th, so I was hoping that the money in the escrow would be sent to  `destination` on midnight yesterday.

Reply:
There is only Cron integration for tally execution so far. If you want to return the tokens to the source account you need to send a release message after expiration by any account. 
The `destination` account only makes sense if the source account wants to release tokens via msg. I don't think this is the case in your example. So it can be `0000000000000000000000000000000000000000` as well. I don't think we have a sanity check for this.

Reply:
Thanks for digging into this.  It was just a spike to showcase some of weave's cool features.  I wanted the source to equal the destination in order to indicate that the destination wasn't getting free tokens.